NotificationBox/README.md

```diff
@@ -0,0 +1,64 @@
+NotificationBox MailExtension Experiment
+========================================
+
+The NotificationBox Experiment allows to show notifications inside Thunderbird.
+
+| Item          | Value
+| ------------- | --------
+| Description   | Experiment to show notifications inside Thunderbird
+| Status        | Draft
+| Compatibility | Thunderbird 128
+| Tracking      | [bug 1674002](https://bugzilla.mozilla.org/show_bug.cgi?id=1674002)
+
+Usage
+-----
+
+Add the NotificationBox Experiment to your add-on. Your `manifest.json` needs an entry like this:
+
+```
+"experiment_apis": {
+    "NotificationBox": {
+      "schema": "NotificationBox/schema.json",
+      "parent": {
+        "scopes": [
+          "addon_parent"
+        ],
+        "paths": [
+          [
+            "NotificationBox"
+          ]
+        ],
```

Comment:
```suggestion
      "parent": {
        "scopes": ["addon_parent"],
        "paths": [["NotificationBox"]],
```

Optional I guess, but I find the json formatted syntax to be quite verbose, this is a bit more compact.